backend/core/src/email/email.service.spec.ts

```diff
@@ -143,6 +173,22 @@ describe("EmailService", () => {
         },
         t: {
           hello: "Hello",
+          seeListing: "See Listing",
+        },
+      },
+    })
+
+    await translationsService.create({
+      jurisdiction: {
+        id: jurisdiction.id,
+      },
+      language: Language.es,
+      translations: {
+        confirmation: {
+          yourConfirmationNumber: "SPANISH NUMBER", // UPDATED
```

Comment:
It seems like we likely don't want this string and the one on 191 to include `SPANISH` ?